src/alert-manager/config/alert_manager.py

```diff
@@ -1,6 +1,15 @@
 #!/usr/bin/env python
 
 import copy
+from flatten_dict import flatten, unflatten
+
+def soft_update_dict(dict_0, dict_1):
+    """
+    keep original key in nested dict if not present in the new dict
+    """
+    res = flatten(dict_0)
+    res.update(flatten(dict_1))
+    return unflatten(res)
```

Comment:
How about use this solution: https://stackoverflow.com/questions/3232943/update-value-of-a-nested-dictionary-of-varying-depth instead of introduce a new lib

Reply:
> How about use this solution: https://stackoverflow.com/questions/3232943/update-value-of-a-nested-dictionary-of-varying-depth instead of introduce a new lib

Updated, thanks